tests/test_matrix.py

```diff
@@ -92,6 +92,61 @@ def test_addMatrix():
     matSum = addMatrix(matA, matB)
     assert matSum.__str__() == "[{x}+{y}+{1.0},2{x}^{2.0};{5.0},2{x}{y}-{1.0}]"
 
+def test_scalarAddMatrix():
+
+    mat = getTokens("[1, 2; \
+                    2,  1]")
+    const = 2
+    matSum = scalarAdd(const, mat)
+    assert matSum.__str__() == "[{3.0},{2.0};{2.0},{3.0}]"
+
+    mat = getTokens("[1, 2, 3;\
+                        4, 5, 6;\
+                        7, 8, 9]")
+    const = 3
+    matSum = scalarAdd(const, mat)
+    assert matSum.__str__() == "[{4.0},{2.0},{3.0};{4.0},{8.0},{6.0};{7.0},{8.0},{12.0}]"
+
+def test_scalarSubMatrix():
+
+    mat = getTokens("[8, 6]")
```

Comment:
Change the test case to some 2x2 matrix here, since you have defined `scalarSub` as **{mat} - {lambda}{identity mat}**.